pkg/migrations/op_add_column_test.go

```diff
@@ -786,6 +786,88 @@ func TestAddColumnWithUpSql(t *testing.T) {
 				TriggerMustNotExist(t, db, schema, "products", triggerName)
 			},
 		},
+		{
+			name: "add column with up sql missing parentheses",
+			migrations: []migrations.Migration{
+				{
+					Name: "01_add_table",
+					Operations: migrations.Operations{
+						&migrations.OpCreateTable{
+							Name: "products",
+							Columns: []migrations.Column{
+								{
+									Name: "id",
+									Type: "serial",
+									Pk:   ptr(true),
+								},
+								{
+									Name:   "name",
+									Type:   "varchar(255)",
+									Unique: ptr(true),
+								},
+							},
+						},
+					},
+				},
+				{
+					Name: "02_add_column",
+					Operations: migrations.Operations{
+						&migrations.OpAddColumn{
+							Table: "products",
+							Up:    "select name",
```

Comment:
I believe we also want to test just `name` here? ie
```suggestion
							Up:    "name",
```

Reply:
I'll add a separate test for that. The one with `select` definitely failed without `( )`

Reply:
Done and looks fine